### ISSUE TITLE
[Target Request] TMOTORF7

### ISSUE THREAD

**Issue by StuartD89** [emuflight/EmuFlight#810]
T-Motor F7 HD 
Target: TMOTORF7

https://store.tmotor.com/goods.php?id=935

[Apex - RCInpower 2306.5 1880kv.txt](https://github.com/emuflight/EmuFlight/files/8758099/Apex.-.RCInpower.2306.5.1880kv.txt)

**Comment by nerdCopter** [comment]
this one doesn't work? 
![image](https://user-images.githubusercontent.com/56646290/170037185-98616eb9-fb95-4f61-ae11-27d61e494d21.png)

**Comment by StuartD89** [comment]
Hello,

No it does not, I tried it hoping that it would still work but it just locked my flight controller out, I had to start it up in boot mode to flash betaflight back on it and on betaflight it uses the TMOTORF7 target.

On 24 May 2022, at 13:42, nerdCopter ***@***.***> wrote:

﻿

this one doesn't work?
[image]<https://user-images.githubusercontent.com/56646290/170037185-98616eb9-fb95-4f61-ae11-27d61e494d21.png>

**Comment by BeauBrewski** [comment]
I have an analog build that uses this target. Flashed `TMOTORF7` (the 0.3.4 version), gyros and everything worked. Flashed `TMTR-TMOTORF7` (0.4.0 version) gyros and everything worked. Even flashed `TMOTORF7V2` and gyros and booting worked (however other resources are definitely wrong as incorrect target). 

Unable to recreate above problem. 
Being HD build should not change anything except how you set up ports and OSD.

**Comment by StuartD89** [comment]
I will try to flash again later, when I flashed it the first time the computer wouldn’t even recognise the USB port. I used boot mode to get it back.

On 24 May 2022, at 23:26, BeauBrewski ***@***.***> wrote:

﻿

I have an analog build that uses this target. Flashed TMOTORF7 (the 0.3.4 version), gyros and everything worked. Flashed TMTR-TMOTORF7 (0.4.0 version) gyros and everything worked. Even flashed TMOTORF7V2 and gyros and booting worked (however other resources are definitely wrong as incorrect target).

Unable to recreate above problem.
Being HD build should not change anything except how you set up ports and OSD.
